mopidy_mpd/network.py

```diff
@@ -507,5 +507,9 @@ def send_lines(self, lines):
         if not lines:
             return
 
+        # using translate() and fromkeys() to remove all control characters
+        mapping =  dict.fromkeys(range(32))
```

Comment:
I think the mapping should be a module-level constant to not recreate it on each request. Also, using `ord(" ")` instead of `32` or mentioning that this removes the first 32 characters in the ASCII encoding would be helpful.